pkg/local-gadget-manager/local-gadget-manager.go

```diff
@@ -270,6 +270,20 @@ func (l *LocalGadgetManager) PublishEvent(tracerID string, line string) error {
 	return nil
 }
 
+func (l *LocalGadgetManager) TracerAddTrace(containerSelector containercollection.ContainerSelector) (string, error) {
+	// We are not running multiple tracers per instance so the tracer ID doesn't
+	// need to be unique.
+	tracerID := "local_gadget_tracer_id"
+	if err := l.tracerCollection.AddTracer(tracerID, containerSelector); err != nil {
+		return "", err
+	}
+	return tracerID, nil
+}
+
+func (l *LocalGadgetManager) TracerRemoveTrace(tracerID string) error {
+	return l.tracerCollection.RemoveTracer(tracerID)
+}
+
```

Comment:
I find the name of these two functions a bit difficult to follow, but I know it's because `AddTracer` and `Delete` are already taken and used for other things. Maybe we can put a comment indicating why these functions exist and saying we could rename them once `AddTracer` and `Delete` aren't needed anymore?

Another possibility is to rename `AddTracer` to `AddTracerLegacy`, and `Delete` to `RemoveTracerLegacy` and use `AddTracer` and `RemoveTracer` here? (seems a bit more invasive but maybe it's the best option for the long term).